srcpkgs/python3-ldap/template

```diff
@@ -13,7 +13,7 @@ maintainer="Michael Aldridge <maldridge@voidlinux.org>"
 license="Python-2.0"
 homepage="https://www.python-ldap.org"
 distfiles="${PYPI_SITE}/p/${_pkgname}/${_pkgname}-${version}.tar.gz"
-checksum=7d1c4b15375a533564aad3d3deade789221e450052b21ebb9720fb822eccdb8e
+checksum=ab26c519a0ef2a443a2a10391fa3c5cb52d7871323399db949ebfaa9f25ee2a0
 
 pre_build() {
 	sed -i "s:^libs = .*:libs = ldap_r lber sasl2 ssl crypto:" setup.cfg
```

Comment:
can we switch to vsed here?

Reply:
I actually don't think that expression changes anything here. openldap 2.6 does not provide `ldap_r` anymore, it's `ldap`. I have tried removing the pre_build and it builds and checks fine still.